build-tools/packages/build-tools/src/fluidBuild/tasks/leaf/biomeTasks.ts

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import path from "path";
+import { GitRepo } from "../../../common/gitRepo";
+import { LeafWithFileStatDoneFileTask } from "./leafTask";
+import { getResolvedFluidRoot } from "../../../common/fluidUtils";
+import findUp from "find-up";
+
+export class BiomeTask extends LeafWithFileStatDoneFileTask {
+	private repoRoot: string | undefined;
+	private gitRepo: GitRepo | undefined;
+
+	/**
+	 * Includes all files in the task's package directory and any biome config files in the directory tree. Files ignored
+	 * by git are excluded.
+	 */
+	protected async getInputFiles(): Promise<string[]> {
+		this.repoRoot ??= await getResolvedFluidRoot(true);
+		this.gitRepo ??= new GitRepo(this.repoRoot);
```

Comment:
Possibly someday consider if 'repoRoot' and 'gitRepo' should be lazily initialized singletons.

(No change requested).